web/styles/settings.css

```diff
@@ -1406,6 +1406,16 @@ $option_title_width: 180px;
             }
         }
 
+        .collapse-settings-btn,
+        .collapse-settings-btn > .icon {
+            color: hsl(200deg 100% 40%);
```

Comment:
It would be nice to future-proof this with CSS variables for the colors.

Reply:
Pushed a change, but the hover color is a bit different now, seeking confirmation [here](https://chat.zulip.org/#narrow/stream/101-design/topic/org.20settings.20-.20show.20more.20-.20icon.20alignment/near/1935956)

Also gonna open an issue for standardising these colors as described in [this message](https://chat.zulip.org/#narrow/stream/101-design/topic/org.20settings.20-.20show.20more.20-.20icon.20alignment/near/1935956).